Artifacts.toml

```diff
@@ -0,0 +1,39 @@
+[densenet121]
+git-tree-sha1 = "ffc7f7ed1e7f67baca4b76f6c100e0d5042ff063"
+lazy = true
+
+    [[densenet121.download]]
+    sha256 = "3fd10f0be70cf072fa7f1358f1fbbe01138440dbcaec1b7c8e007084382c1557"
+    url = "https://github.com/darsnack/MetalheadWeights/releases/download/v0.1.1/densenet121-0.1.1.tar.gz"
```

Comment:
Will this repo be moved into FluxML or incorporated into Metalhead somehow?

Reply:
Done.